### PR TITLE
Updated utils readme

### PR DESCRIPTION
**Pull request by AbuNavsa**
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+This service depends on:
+- No other microservices
+
 # Dev setup
 In order to run the unit tests, setup a virtual env and install requirements
-1. Checkout the code
-1. Setup a venv and activate: `python3 -m venv .venv && source .venv/bin/activate`
-1. Install dev requirements: `pip install --upgrade pip && pip install -r requirements-dev.txt`
+1. [Developer setup guide](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-setup.md)
 1. Install pre-commit hook: `pre-commit install`
```
Comment (srh-sloan) [inline README.md:12]: These pre-commit and testing bits are also covered in the shared workflows, so please link to them :)

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
 # funding-service-design-utils
-Shared library for funding service design apps.
+Utils serves as a shared library for funding service design applications.These utilities are meant to be reused across the project, streamlining our coding process and ensuring the reliability of our funding service design applications.
```
Comment (srh-sloan) [inline README.md:2]: Missing a space after applications.

**Review by srh-sloan — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ This service depends on:
 # Dev setup
```
Comment (srh-sloan) [inline README.md:9]: Can you amend this whole section to reference the shared readmes please? So for dev setup, testing, pre-commit. But keep the note about adding stuff to `pyrpoject.toml` somewhere as that won't be in shared docs.

**Review by srh-sloan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AbuNavsa) [reply README.md:9]: added the testing read me section now so all of above are linked to corresponding shared readmes

**Review by srh-sloan — DISMISSED** [review]

**Review by srh-sloan — APPROVED** [review]

**Pull request merged**